main.py

```diff
@@ -2,12 +2,21 @@
 from datetime import datetime
 
 
+with open("darabase.txt", "w") as f:    # just to create the file automatically 
```

Comment:
don't know if it's really needed, but it's better that creating this file yourself

Reply:
database